decidim-core/app/assets/stylesheets/decidim/modules/_cards.scss

```diff
@@ -77,11 +77,6 @@ $datetime-bg: $primary;
   @extend .heading5;
 }
 
-.card__title--strong{
```

Comment:
Did you check this is not being used anywhere else?

Reply:
Yes, I've checked and re-checked. Also talked with @carolromero if we would need these on the future or for other components and she said that there isn't any plan to use this style.